### ISSUE TITLE
Using Official Client

### ISSUE THREAD

**Issue by Onalu** [polyfractal/sherlock#87]
As discussed [here](https://github.com/elasticsearch/elasticsearch-php/issues/68), official client will remain as a low level library. And Sherlock's fluent API is very useful and clean. So, It would be good if Sherlock supports official client.

**Comment by polyfractal** [comment]
Sorry for the delay, got caught up with other stuff at work.

You mentioned in the other thread that you were already modifying Sherlock to integrate the official client?  I won't have time to do the integration myself (and Sherlock has already suffered a lot of bit-rot because of my negligence).  But I would definitely help provide guidance on a PR to get it integrated, and help out whenever I have time.

**Comment by Onalu** [comment]
Yes, I was working on search requests. Right now, I have some other things in my hand but I will begin to work on this integration, again. After I have done with search request, I can make a PR and you can comment on that or after I finished all the integration stuff, I can make a PR. I guess I prefer second one, but I am not that sure. Maybe it is a good idea that discussing earlier on a small part.

**Comment by Onalu** [comment]
Sorry, I couldn't spend much time to this job, so it is still incomplete. I've done much of the integration stuff and updated tests.  There are some works such as response classes, API updates etc. So, I did not make a PR because of these unfinished works. You can check [my fork](https://github.com/Onalu/sherlock) and make comments when you are available.

**Comment by erichard** [comment]
Hi, I was looking for a fluent builder for elasticSearch and a fall across this repo and issue. 

Do you think it is still and interesting feature on top of the official client ? Does the work to do after 3 years wouldn't be huge ?
